api/tests/integration/users/test_init_config.py

```diff
@@ -24,3 +29,62 @@ def test_returns_200_when_no_user_exists(db, django_client):
 
     # Then
     assert response.status_code == status.HTTP_200_OK
+
+
+def test_valid_request_creates_a_superuser(db, django_client):
+    # Given
+    url = reverse("api-v1:users:config-init")
+    user_dict = {
```

Comment:
I'd say that `user_dict` is a bit misleading here. We can probably just be generic and call this like `data` or `form_data`.